### PR TITLE
create database

### PR DESCRIPTION
**Pull request by DuaaH**

### CODE REVIEW

**Review by someyaaltous — APPROVED** [review]

**Review by SajaLahaleeh — APPROVED** [review]

File: src/database/db_build.sql [hunk 1]
```diff
@@ -0,0 +1,46 @@
+BEGIN ;
+DROP TABLE IF EXISTS countries CASCADE;
+DROP TABLE IF EXISTS teams CASCADE;
+
+CREATE TABLE countries(
```
Comment (jema28) [inline src/database/db_build.sql:5]: Please add a required field on some of these values e.g. team name

File: src/database/db_build.sql [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+  id_countries SERIAL PRIMARY KEY ,
+  country_name VARCHAR (255) NOT NULL,
+  continent VARCHAR (255) NOT NULL
+);
+CREATE TABLE teams(
+  id SERIAL PRIMARY KEY ,
+  id_countries INT REFERENCES countries(id_countries),
```
Comment (jema28) [inline src/database/db_build.sql]: This should be a foreign key.

**Review by jema28 — CHANGES_REQUESTED** [review]

**Comment by jema28** [comment]
Please correct your **assets** folder name - small spelling error 😺

File: src/database/db_build.sql (hunk 1 above)
Comment (DuaaH) [reply src/database/db_build.sql:5]: @jema28 we add a required fields

File: src/database/db_build.sql (hunk 2 above)
Comment (DuaaH) [reply src/database/db_build.sql]: @jema28 add foreign key

File: src/database/db_build.sql [hunk 3]
```diff
@@ -9,38 +9,39 @@ CREATE TABLE countries(
 );
 CREATE TABLE teams(
   id SERIAL PRIMARY KEY ,
-  id_countries INT REFERENCES countries(id_countries),
-  team_name TEXT,
+  id_countries int,
+ FOREIGN KEY (id_countries) REFERENCES countries(id_countries),
```
Comment (jema28) [inline src/database/db_build.sql]: Just a small style error, needs to line up with the other fields. Can you add eslint to the project? And always eslint before you push up.

**Review by jema28 — CHANGES_REQUESTED** [review]

File: src/database/db_build.sql (hunk 3 above)
Comment (jema28) [reply src/database/db_build.sql]: You can add eslint in another PR.

Comment (DuaaH) [reply src/database/db_build.sql]: @jema28  added eslint

**Review by jema28 — COMMENTED** [review]
Eslint not working, should be fixed in another PR. Will merge database though.

**Review by jema28 — APPROVED** [review]

**Pull request merged**